### DISCUSSION TITLE
ALT key with Whites and Blacks

### DISCUSSION THREAD

**Discussion by Dehazeme** [rsjaffe/MIDI2LR#571] [Q&A] [answered]
Holding the ALT key while setting Whites and Blacks displays the clipping limits. Can this be functionality be assigned to an encoder?
Thanks.

**Comment by rsjaffe** [comment, marked as answer]
No, it can't. I requested that function long ago, and Adobe has ignored the request. Additionally, holding the alt key down while adjusting whites/blacks in MIDI2LR doesn't work. That's a Lightroom bug that was reported years ago. See the FAQ #6 https://github.com/rsjaffe/MIDI2LR/wiki#faqknown-issues.
